Fix #11389: Updates margin of the supplemental card in exploration player page

atpug22 opened the pull request:
## Overview
<!--
READ ME FIRST:
Please answer *both* questions below and check off every point from the Essential Checklist!
If there is no corresponding issue number, fill in N/A where it says [fill_in_number_here] below in 1.
-->

1. This PR fixes #11389.
2. This PR does the following: This added margin so that oppia's response can be clearly seen.

## Essential Checklist

- [x] The PR title starts with "Fix #bugnum: ", followed by a short, clear summary of the changes. (If this PR fixes part of an issue, prefix the title with "Fix part of #bugnum: ...".)
- [x] The linter/Karma presubmit checks have passed locally on your machine.
- [x] "Allow edits from maintainers" is checked. (See [here](https://help.github.com/en/github/collaborating-with-issues-and-pull-requests/allowing-changes-to-a-pull-request-branch-created-from-a-fork) for instructions on how to enable it.)
  - This lets reviewers restart your CircleCI tests for you.
- [x] The PR is made from a branch that's **not** called "develop".

## Proof that changes are correct

https://user-images.githubusercontent.com/56637531/104051253-e0881800-520d-11eb-9a13-a113e51dd67e.mp4


<!--
Add videos/screenshots of the user-facing interface to demonstrate that the changes made in this PR work correctly.
-->

## PR Pointers

- Make sure to follow the [instructions for making a code change](https://github.com/oppia/oppia/wiki/Contributing-code-to-Oppia#instructions-for-making-a-code-change).
- Oppiabot will notify you when you don't add a PR_CHANGELOG label. If you are unable to do so, please @-mention a code owner (who will be in the Reviewers list), or ask on [Gitter](https://gitter.im/oppia/oppia-chat).
- For what code owners will expect, see the [Code Owner's wiki page](https://github.com/oppia/oppia/wiki/Oppia%27s-code-owners-and-checks-to-be-carried-out-by-developers).
- Make sure your PR follows conventions in the [style guide](https://github.com/oppia/oppia/wiki/Coding-style-guide), otherwise this will lead to review delays.
- Never force push. If you do, your PR will be closed.
- Oppiabot can assign anyone for review/help if you leave a comment like the following: "{{Question/comment}} @{{reviewer_username}} PTAL"
- Some of the e2e tests are flaky, and can fail for reasons unrelated to your PR. We are working on fixing this, but in the meantime, if you need to restart the tests, please check the ["If your build fails" wiki page](https://github.com/oppia/oppia/wiki/If-your-build-fails).

atpug22 committed:
adding margin only will fix it

atpug22 committed:
adding margin only will fix it

atpug22 committed:
Merge remote-tracking branch 'upstream/develop' into pencil-code-editor

atpug22 committed:
Merge remote-tracking branch 'upstream/develop' into pencil-code-editor

atpug22 commented:
@vojtechjelinek @kevintab95 @aks681 PTAL.
I guess adding margin only at the end of the page will do the job.

DubeySandeep commented:
@atpug22, @atpug22 Can you please update the PR title as asked in the checklist?

atpug22 commented:
> @atpug22, @atpug22 Can you please update the PR title as asked in the checklist?

Done

vojtechjelinek commented:
@atpug22 Thanks a lot for a quick fix. Please in the top message _Proof that changes are correct_ section add screenshots that show that it was really fixed. Preferable before and after screenshots on multiple resolutions.

atpug22 commented:
> @atpug22 Thanks a lot for a quick fix. Please in the top message _Proof that changes are correct_ section add screenshots that show that it was really fixed. Preferable before and after screenshots on multiple resolutions.

Done @vojtechjelinek

atpug22 committed:
Merge remote-tracking branch 'upstream/develop' into pencil-code-editor

kevintab95 reviewed (approved):
LGTM, thanks @atpug22!

vojtechjelinek commented:
@atpug22 For some reason the lint checks are timeouting, can you please merge from develop?

atpug22 commented:
> @atpug22 For some reason the lint checks are timeouting, can you please merge from develop?
@vojtechjelinek 
Actually, I am having an issue with pushing this code. It is getting stuck after lint checks as shown in the screenshot. Before I force pushed the code with --no-verify and I guess I am having a lint check issue.
![Screenshot 2021-01-08 190846](https://user-images.githubusercontent.com/56637531/104187978-920d9000-543e-11eb-81fc-e54e162467f6.png)
Today I am going to discuss this with @DubeySandeep. After that, I'll Push the code.

vojtechjelinek commented on core/templates/pages/exploration-player-page/learner-experience/conversation-skin.directive.html:
```diff
@@ -276,6 +276,9 @@
 
+  .conversation-skin-supplemental-card-container {
+    margin-bottom: 200px;
+  }
```
Could you try to insert an empty line below?

vojtechjelinek commented:
@atpug22 That is weird, the code seems completely fine to me, added one suggestion that could help. cc @DubeySandeep seems like some linting error

atpug22 commented:
> @atpug22 That is weird, the code seems completely fine to me, added one suggestion that could help. cc @DubeySandeep seems like some linting error

@vojtechjelinek  Yes, we tried that but still, it wasn't working.

vojtechjelinek commented:
> > @atpug22 That is weird, the code seems completely fine to me, added one suggestion that could help. cc @DubeySandeep seems like some linting error
> 
> @vojtechjelinek Yes, we tried that but still, it wasn't working.

I still thing the empty line should be added though no matter what :smile:, but yeah I'm not sure how to fix this.

atpug22 commented:
> > > @atpug22 That is weird, the code seems completely fine to me, added one suggestion that could help. cc @DubeySandeep seems like some linting error
> > 
> > @vojtechjelinek Yes, we tried that but still, it wasn't working.
> 
> I still thing the empty line should be added though no matter what :smile:, but yeah I'm not sure how to fix this.

Yes I added that and will push the code once the issue resolves.

DubeySandeep commented on core/templates/pages/exploration-player-page/learner-experience/conversation-skin.directive.html:
```diff
@@ -276,6 +276,9 @@
   }
 
+  .conversation-skin-supplemental-card-container {
+    margin-bottom: 200px;
```
Can we add this to the existing block on line 298?

atpug22 replied:
But that block is under the media query for width<=360px but we need this margin for width greater than that.

DubeySandeep replied:
I think it's not inside the media query block, can you please check again? :)

atpug22 replied:
ok. Sorry I didn't check that before. I thought that after that all the media-query blocks are there so I wrote before first media query block.
Now it worked and I pushed the code. PTAL Thanks.
@DubeySandeep

vojtechjelinek replied:
@DubeySandeep I think there is the issue that lint tests timeout instead of producing an error for this issue.

DubeySandeep replied:
@vojtechjelinek, The issue is resolved in #11672 :)

vojtechjelinek replied:
Great, thanks a lot.

atpug22 committed:
merged two css styling of same class into one

atpug22 committed:
merged two css styling of same class into one

atpug22 committed:
changing order in css class
